src/elements/OcMenuItem.vue

```diff
@@ -1,14 +1,12 @@
 <template>
-  <li @click="onClick">
-    <a href="#">
-      <div v-if="icon" class="uk-flex uk-flex-middle">
-        <oc-icon :name="icon" class="uk-margin-small-right" />
-        <slot name="default" />
-      </div>
-      <template v-else>
-        <slot name="default" />
-      </template>
-    </a>
+  <li class="oc-menu-item" @click="$_ocMenuItem_onClick">
+    <div v-if="icon" class="uk-flex uk-flex-middle">
```

Comment:
We need to have the icon wrapped in the anchor tag, so it becomes clickable as well.